src/main/java/org/znerd/confluence/client/http/ConfluenceRestClient.java

```diff
@@ -426,6 +444,31 @@ public void deleteLabelFromPage(String contentId, String labelName) {
         sendRequest(deleteLabelFromPageRequest, (ignored) -> null);
     }
     
+    public ArrayList<ConfluenceLabel> getLabelsFromPage(String contentId) {
```

Comment:
Some comments/questions:
- Should this method also be defined in the parent interface `ConfluenceClient`? I would expect it should.
- It would be better to change the declaration to return `List` instead of `ArrayList`, to hide that implementation detail.